src/main/java/mate/model/Driver.java

```diff
@@ -54,4 +78,14 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(id, name, licenseNumber);
     }
+
```

Comment:
Maybe we should add login into equals and hashcode?

Reply:
Done